pkgs/by-name/tr/tre/package.nix

```diff
@@ -1,30 +1,56 @@
 {
   lib,
   stdenv,
-  fetchurl,
-  fetchpatch,
+  fetchFromGitHub,
+  autoreconfHook,
+  automake,
+  darwin,
+  gettext,
+  libiconv,
+  libtool,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "tre";
-  version = "0.8.0";
-  src = fetchurl {
-    url = "https://laurikari.net/tre/${pname}-${version}.tar.gz";
-    sha256 = "1pd7qsa7vc3ybdc6h2gr4pm9inypjysf92kab945gg4qa6jp11my";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "laurikari";
+    repo = "tre";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-5O8yqzv+SR8x0X7GtC2Pjo94gp0799M2Va8wJ4EKyf8=";
   };
 
-  patches = [
-    (fetchpatch {
-      url = "https://sources.debian.net/data/main/t/tre/0.8.0-6/debian/patches/03-cve-2016-8859";
-      sha256 = "0navhizym6qxd4gngrsslbij8x9r3s67p1jzzhvsnq6ky49j7w3p";
-    })
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    automake
+    libtool
   ];
 
+  buildInputs =
+    [
+      gettext
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.apple_sdk.frameworks.CoreFoundation
```

Comment:
```suggestion
```
As far as I know this is automatically available. Did you get an error without it?

Reply:
I thought I needed this, since it was part of the output of `otool`, but no errors occur. Removed in latest commit.